### DISCUSSION TITLE
FPS Issue in WPF when Using CefSharp

### DISCUSSION THREAD

**Discussion by CodigosPruebas367** [cefsharp/CefSharp#4623] [General]
Hello, 
In the demos available here, you can test it yourself. Using WinForms, the FPS exceeds 60, but in WPF it's capped at 30. I've tried everything, including disable-gpu, and all the solutions I found when searching for this issue.

Do you know if it's possible to make it work at more than 30 FPS in WPF?

I'm checking the FPS by visiting [www.testufo.com](http://www.testufo.com/)

**Comment by amaitland** [comment]
> Using WinForms, the FPS exceeds 60

The like for like comparison is https://github.com/cefsharp/CefSharp.Wpf.HwndHost#readme

See https://github.com/cefsharp/CefSharp/wiki/General-Usage#offscreen-rendering-osr to understand how the WPF implementation works.

> but in WPF it's capped at 30

The default is 30, this is expected.

> including disable-gpu, and all the solutions I found when searching for this issue.

Leave GPU acceleration enabled and make sure you have an app.manifest as per https://github.com/cefsharp/CefSharp/wiki/Quick-Start-For-MS-.Net-4.x#40-add-appmanifest-to-your-application

You can control the frame rate via https://cefsharp.github.io/api/118.6.x/html/P_CefSharp_BrowserSettings_WindowlessFrameRate.htm
